src/features/defi/helpers/utils.ts

```diff
@@ -46,15 +46,11 @@ export const canCoverTxFees = ({
 
 export const getFeeDataFromEstimate = ({
   average,
-  fast,
 }: FeeDataEstimate<EvmChainId>): FeeData<EvmChainId> => ({
-  txFee: BigNumber.max(
-    average.txFee,
-    bnOrZero(bn(fast.chainSpecific.gasPrice).times(average.chainSpecific.gasLimit)),
-  ).toFixed(0), // use worst case average eip1559 vs fast legacy
+  txFee: average.txFee,
   chainSpecific: {
-    gasLimit: average.chainSpecific.gasLimit, // average and fast gasLimit values are the same
-    gasPrice: fast.chainSpecific.gasPrice, // fast gas price since it is underestimated currently
+    gasLimit: average.chainSpecific.gasLimit,
+    gasPrice: average.chainSpecific.gasPrice,
```

Comment:
we can just use average across the board now since it will be a viable `gasPrice` instead of the low balled zrx gas estimate